Makefile

```diff
@@ -4,25 +4,31 @@ GROCY_VERSION = v3.3.0
 COMPOSER_VERSION = 2.1.5
 COMPOSER_CHECKSUM = be95557cc36eeb82da0f4340a469bad56b57f742d2891892dcb2f8b0179790ec
 IMAGE_TAG ?= $(shell git describe --tags --match 'v*' --dirty)
-
 IMAGE_PREFIX ?= docker.io/grocy
+POD_NAME ?= grocy-pod
+APP_DB_VOLUME_NAME ?= grocy-app-db
+PUBLISH_AT ?= 127.0.0.1:8080
+
 PLATFORM ?= linux/386 linux/amd64 linux/arm/v6 linux/arm/v7 linux/arm64/v8 linux/ppc64le linux/s390x
 
 build: manifest
 
 create: pod
 	podman create \
+        --add-host grocy:127.0.0.1 \
+        --add-host frontend:127.0.0.1 \
+        --add-host backend:127.0.0.1 \
```

Comment:
What are these hostname references used for?

(the only service-to-service connectivity I can think of at the moment is that the `frontend` NGINX instance is [configured](https://github.com/grocy/grocy-docker/blob/2a0d43f11d86af2c9f18d2398b9ced194d16a2fb/docker_frontend/common.conf#L12) to pass some requests through to the `backend` service.. very possible I've forgotten something)